frontend/src/ui/Blocks/Series.tsx

```diff
@@ -193,46 +210,364 @@ type SeriesBlockContainerProps = {
+        </>,
+        order: () => <>
+            <div onClick={() => close()}>{t("series.settings.order")}</div>
+            <li tabIndex={0} onClick={() => {
+                setEventOrder("NEW_TO_OLD");
+                close();
+            }}>{t("series.settings.new-to-old")}</li>
+            <li tabIndex={0} onClick={() => {
+                setEventOrder("OLD_TO_NEW");
+                close();
+            }}>{t("series.settings.old-to-new")}</li>
+        </>,
+    });
+
+    return (
+        <Floating hideArrowTip borderWidth={0} css={listStyle}>
+            <ul>{list}</ul>
+        </Floating>
+    );
+};
+
+
+const SliderView: React.FC<{ children: ReactNode }> = ({ children }) => {
+    const ref = useRef<HTMLDivElement>(null);
+    const scrollDistance = 240;
+
+    const [rightVisible, setRightVisible] = useState(false);
+    const [leftVisible, setLeftVisible] = useState(false);
+
+    const setVisibilities = () => {
+        if (ref.current) {
+            setRightVisible(ref.current.scrollLeft
+                    + ref.current.offsetWidth
+                    < (ref.current.scrollWidth - 16));
+            setLeftVisible(ref.current.scrollLeft > 16);
+        }
+    };
+
+    const scroll = (distance: number) => {
+        if (ref.current) {
+            ref.current.scrollLeft += distance;
+            setVisibilities();
+        }
+    };
+
+    useEffect(() => {
+        setVisibilities();
+    }, []);
+
+    const buttonCss = {
+        alignSelf: "center",
+        backgroundColor: "var(--grey65)",
+        borderRadius: 50,
+        padding: 11,
+        transition: "all .1s ease-in",
+        svg: {
+            color: "white",
+            display: "block",
+            fontSize: 26,
+        },
+        ":hover, :focus": {
+            backgroundColor: "var(--grey40)",
+        },
+        ":focus-visible": {
+            outline: "2px solid var(--accent-color)",
+        },
+    };
+
+    return (
+        <div css={{ position: "relative" }}>
+            <div tabIndex={-1} onScroll={() => setVisibilities()} ref={ref} css={{
+                display: "grid",
+                gridAutoFlow: "column",
+                gridAutoColumns: 230,
+                gap: 16,
+                overflowX: "auto",
+                scrollBehavior: "smooth",
+                scrollSnapType: "inline mandatory",
+                "> *": {
+                    scrollSnapAlign: "start",
+                },
+            }}>
+                {children}
+                {leftVisible && <ProtoButton
+                    onClick={() => scroll(-scrollDistance)}
+                    css={{ position: "absolute", left: 0, ...buttonCss }}>
+                    <FiChevronLeft />
+                </ProtoButton>}
+                {rightVisible && <ProtoButton
+                    onClick={() => scroll(scrollDistance)}
+                    css={{ position: "absolute", right: 0, ...buttonCss }}>
```

Comment:
Also: I would use `right: 8` or sth like that to move the buttons a bit away from the container edge. Looks better IMO.